README.md

```diff
@@ -200,6 +201,35 @@ $ git fetch
 $ git checkout v0.3.0
 ~~~
 
+If you've [installed via Homebrew](#homebrew-on-mac-os-x), then upgrade
+via its `brew` command:
+
+~~~ sh
+$ brew update
+$ brew upgrade jawshooah/nodenv/nodenv node-build
```

Comment:
You're right, a head-only formula doesn't work with `brew upgrade`. I believe I wrote this section in anticipation of a `node-build` release.